internal/pkg/cli/svc_package.go

```diff
@@ -150,6 +160,34 @@ func newWorkloadStackGenerator(o *packageSvcOpts) (workloadStackGenerator, error
 	return deployer, nil
 }
 
+func newTemplateOverrider(pathToOverridesDir, app, env string, fs afero.Fs, provider *sessions.Provider) (clideploy.Overrider, error) {
+	info, err := override.Lookup(pathToOverridesDir, fs)
+	if err != nil {
+		var errNotExist *override.ErrNotExist
+		if errors.As(err, &errNotExist) {
+			return new(override.Noop), nil
+		}
+		return nil, fmt.Errorf("look up overrider at %q: %w", pathToOverridesDir, err)
+	}
+	switch {
+	case info.IsCDK():
+		provider.UserAgentExtras("override cdk")
+		return override.WithCDK(pathToOverridesDir, override.CDKOpts{
+			Stdout: log.OutputWriter,
+			FS:     fs,
+			EnvVars: map[string]string{
+				"COPILOT_APPLICATION_NAME": app,
+				"COPILOT_ENVIRONMENT_NAME": env,
+			},
+		}), nil
+	case info.IsYAMLPatch():
+		provider.UserAgentExtras("override yamlpatch")
+		return new(override.Noop), nil
```

Comment:
Just double check: this is a todo right?

Reply:
yea that's a todo 😅